torchtitan/experiments/deepseek_v3/requirements.txt

```diff
@@ -0,0 +1,5 @@
+transformers
+accelerate
+torchdata >= 0.8.0
+datasets >= 2.21.0
+tomli >= 1.1.0 ; python_version < "3.11"
```

Comment:
Are these three added by coincident?

Reply:
I'm planning to add toml support for the training loop and also C4 dataset, so I wanted to ensure we match Titan in terms of base requirements.